packages/wrangler/src/api/startDevWorker/ProxyController.ts

```diff
@@ -471,28 +471,22 @@ export class ProxyController extends Controller<ProxyControllerEventMap> {
 			"Trying to handle inspector message when inspector is disabled"
 		);
```

Comment:
So it appears that this assertion holds even if `this.latestConfig?.experimental?.tailLogs` is true.

When would we still get these inspector messages when the tailLogs feature is turned on? And why do we not check this flag for `Runtime.exceptionThrown` messages?